README.md

```diff
@@ -4,6 +4,10 @@ DevDocs combines multiple developer documentations in a clean and organized web
 
 DevDocs was created by [Thibaut Courouble](https://thibaut.me) and is operated by [freeCodeCamp](https://www.freecodecamp.org).
 
+## We are currently searching for maintainers
+
+Please reach out to `@Thibaut` on [Gitter](https://gitter.im/FreeCodeCamp/DevDocs) if you would like to join the team!
```

Comment:
I would change this to "Please reach out to _the community_ on Gitter if [...]" since I'm technically not the owner anymore, and I don't have all the admin rights to set someone up. But if a group of maintainers can organize themselves on Gitter, I'm happy to help get everyone set up and share context.